documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -437,6 +437,21 @@ method. Specifically, these annotations will be found if they are directly prese
 indirectly present, or meta-present on a given element.
 ====
 
+[[writing-tests-conditional-execution-custom]]
+==== Custom Conditions
+
+A container or test may be enabled or disabled based on the boolean return of a static
+method via the `{EnabledIf}` and `{DisabledIf}` annotations. The method is provided to
```

Comment:
These attributes need to be defined in `documentation/src/docs/asciidoc/link-attributes.adoc`.

Reply:
Didn't noticed this file, thanks! 👍